### PR TITLE
Add Contributing section to README

### LINKED ISSUE #226: Add contributing section to README
I'm writing a much more descriptive guide on contributing to the site for the README.

### PR DESCRIPTION
**Pull request by ethanmad**
# Advanced HW 7 submission

Closes #226.

Name: Ethan Madison
Uniqname: ethanmad

The previous documentation for contributing to the site was lacking
information included in Advanced Exercise - Week 7's specifications in
additon to information missing from that file.

New contributors should have all of the information they need to quickly
make a contribution now.

Signed-off-by: Ethan Madison ethan@ethanmad.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,40 @@
...
+For more information on setting up, see [GitHub's guide][gh docs] or [Jekyll's documentation][jekyll docs].
+
+After making a change and verifying that it works, please submit a [pull request][].
 
 ---------------------
 
 <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br /><span xmlns:dct="http://purl.org/dc/terms/" property="dct:title">Computing for Computer Scientists</span> by <a xmlns:cc="http://creativecommons.org/ns#" href="http://patpannuto.com" property="cc:attributionName" rel="cc:attributionURL">Pat Pannuto</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
+
+
+[GitHub Pages]: https://pages.github.com/
```
Comment (mterwill) [inline README.md]: Great trick! Did not know about this syntax.

**Comment by ethanmad** [comment]
I'd be happy to make the change. I agree, numbering the lines would be much more readable.

Would you like it amended into the previous commit or as a separate commit (for you to squash, presumably)?

**Comment by mterwill** [comment]
Your call!

**Comment by ethanmad** [comment]
Amended the changes into c879cc.  I fixed those lines as requested and made some more changes:

- Added instructions for package managers besides `apt-get`
- Removed the `zlib1g-dev` package from installation instructions. I don't see what it's required for, and if it is a dependency, the package manager should pull it in. It doesn't exist in the repos for my platform, anyhow.

Should be all good to go now!

**Comment by mterwill** [comment]
I'm good with this – thanks @ethanmad! Merging + giving you credit.

**Pull request merged**